### PR TITLE
[#45043] Add SCHEMA Modifications to on-premise restore docs

### PR DESCRIPTION
**Pull request by ghost**
[#45043] Add SCHEMA Modifications to on-premise restore docs

https://community.openproject.org/work_packages/45043

### CODE REVIEW

**Comment by ghost** [comment]
@machisuji could you please see if the SCHEMA hints are good to go public? It might reduce technical support load, which seems to be a goal :)

File: docs/installation-and-operations/operation/restoring/README.md [hunk 1]
```diff
@@ -241,3 +260,41 @@ You may need to create the `files` directory if it doesn't exist yet.
...
+```
+
+3. The schema with "name of a long number" needs to be renamed after import to `public`.
+
+```psql
+openproject=# ALTER SCHEMA "123456789_1234567_1234567a_123b_12c3_1234_c2a1a123c123" RENAME TO "public";
+ALTER SCHEMA
+```
+
+4. OPTIONAL: The `protocol` in the `settings` table could be changed from `https`to `http`. (This is only recommended if you do not like to use HTTPS)
```
Comment (oliverguenther) [inline docs/installation-and-operations/operation/restoring/README.md]: These steps are no longer valid, as the `protocol` setting has been removed in 12.2

The setting is called `https` and gets values 'true' and 'false'.

**Review by oliverguenther — CHANGES_REQUESTED** [review]

File: docs/installation-and-operations/operation/restoring/README.md (hunk 1 above)
Comment (ghost) [reply docs/installation-and-operations/operation/restoring/README.md]: i removed the OPTIONAL part in order have less information because it is off topic

**Comment by ghost** [comment]
@machisuji please could you check the changes?

**Review by machisuji — APPROVED** [review]
Thanks @ant0nwax!

**Review by machisuji — APPROVED** [review]
👍

**Comment by machisuji** [comment]
I wasn't able to merge this PR due to a big (maybe because you had deleted your user already). I've applied the changes manually in [6f3469](https://github.com/opf/openproject/commit/6f346983a740270179749bd799c8fb53c29e066e). Thanks for your contribution.

**Pull request closed without merging**